libs/sdk-backend-spi/src/workspace/execution/results.ts

```diff
@@ -163,6 +164,29 @@ export interface IResultTotalHeader {
  */
 export type IResultHeader = IResultAttributeHeader | IResultMeasureHeader | IResultTotalHeader;
 
+/**
+ * Represents an execution result warning.
+ * (e.g. when execution was executed successfully, but backend didn't take something into the account)
+ *
+ * @public
+ */
+export interface IResultWarning {
+    /**
+     * Unique identifier of the execution warning
+     */
+    warningCode: string;
+
+    /**
+     * Human readable representation of the execution warning
+     */
+    message: string;
+
+    /**
+     * Execution warning parameters (e.g. when filter was not applied - its ObjRef)
+     */
+    parameters?: (ObjRef | boolean | number | string | null)[];
```

Comment:
a slightly more detail here. does the message contain placeholders ('%s') which should be replaced by client with the values of parameters?